### PR TITLE
Update de.po

### PR DESCRIPTION
**Pull request by stefanleh**
Improved german translations

### CODE REVIEW

**Comment by marusak** [comment]
Can you please provide some explanation to this?
But the same as in #12206 this should be done through zanata.

File: po/de.po [hunk 1]
```diff
@@ -29,34 +29,34 @@ msgstr ""
 msgid " (shared with the OS)"
 msgstr " (gemeinsam mit dem Betriebssystem freigegeben)"
 
+#: pkg/kubernetes/views/node-delete.html:8
+msgid " 1\"Do you want to delete the following Nodes?"
+msgstr "1\"Sind Sie sicher, dass Sie die folgenden Nodes löschen möchten?"
```
Comment (martinpitt) [inline po/de.po]: Sorry, kubernetes and ovirt got removed from the upstream code base. I'll send a PR to refresh translations, so that they get cleaned up. Zanata does that automatically, as the PO template is up to date. As such I'd *really* suggest to update things right here: https://fedora.zanata.org/iteration/view/cockpit/master.

File: po/de.po [hunk 2]
```diff
@@ -3291,32 +3910,37 @@ msgstr "Interner Fehler"
 msgid "Invalid address $0"
 msgstr "Ungültiger Schlüssel"
 
-#: pkg/systemd/host.js:1452 pkg/systemd/shutdown.js:143
+#: pkg/subscriptions/subscriptions-client.js:195
+#, fuzzy
```
Comment (martinpitt) [inline po/de.po]: Why did you add the fuzzy markers to a lot of your new translations? They won't be used like this, just suggested as a starting point in translation UIs.

**Comment by stefanleh** [comment]
Im not quite sure what happened here, i've just changed some translation of Loglevels which were inconsistent. I've used the Github feature for that (no local clone).
@martinpitt I've just assigned for an fedora account and like to help with german translations. My account name is "stle". Maybe you can add me to the translation group for german in zanata?

**Comment by martinpitt** [comment]
@stefanleh: Added your account, thanks!

**Comment by marusak** [comment]
The wanted changes can be now done in Zanata. Closing

**Comment by stefanleh** [comment]
fyi: I've done the changes in Zanata yesterday
